src/deepsparse/image_classification/__init__.py

```diff
@@ -20,12 +20,23 @@
 
 
 _LOGGER = _logging.getLogger(__name__)
-_Dependency = namedtuple("_Dependency", ["name", "version", "necessary"])
+_Dependency = namedtuple("_Dependency", ["name", "import_name", "version", "necessary"])
```

Comment:
Added this field, cause import name can differ from install name for some libraries, example `opencv-python`